rp_redcap/tasks.py

```diff
@@ -45,6 +47,18 @@ def get_required_fields(self, survey_name, redcap_client):
 
         return required_fields
 
+    def get_choices(self, metadata, field_name):
+        choices = []
+        for field in metadata:
+            if field["field_name"] == field_name:
+                choices = dict(
```

Comment:
If `choices` is an array, I think we should be appending rather than overwriting?

Reply:
Hmm, it should actually be a dict, let me fix that.